data/json/monsters/zed_soldiers.json

```diff
@@ -101,7 +101,38 @@
           "require_targeting_player": false,
           "target_moving_vehicles": true,
           "description": "The %s launches a corrosive dart!",
-          "ranges": [ [ 2, 20, "DEFAULT" ] ]
+          "ranges": [ [ 2, 12, "DEFAULT" ] ]
+        }
+      ],
+      "flags": [ "ACIDPROOF", "ACID_BLOOD" ]
+    }
+  },
+  {
+    "id": "mon_zombie_soldier_acid_3",
+    "type": "MONSTER",
+    "name": { "str": "acidic sniper zombie" },
+    "description": "This rotting soldier stands and walks perfectly straight and rigidly, its many, many eyes darting every which way and in every direction. It would be a mere curiosity, if not for the pitch black protrusion where its face used to be; dripping a steaming, caustic bile.",
```

Comment:
```suggestion
    "description": "This rotting soldier stands and walks perfectly straight and rigidly, its many, many eyes darting every which way and in every direction.  It would be a mere curiosity, if not for the pitch black protrusion where its face used to be; dripping a steaming, caustic bile.",
```